catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -46,6 +58,27 @@
 
   public static final String DURATION_METDATA_KEY = "xmpDM:duration";
 
+  public static final String ALTERNATE_MAPPING_OVERRIDE_FILE_KEY = "tika.metadata.mapping.file";
+
+  static final Map<String, List<String>> ALTERNATE_METADATA_KEY_MAPPING;
+
+  static {
+    ALTERNATE_METADATA_KEY_MAPPING = new HashMap<>();
+
+    try (InputStream mappingStream = getAlternateMappings()) {
+      Properties properties = new Properties();
+      properties.load(mappingStream);
+      properties.entrySet().stream()
+          .forEach(
+              entry ->
+                  ALTERNATE_METADATA_KEY_MAPPING.put(
+                      entry.getKey().toString(),
+                      Arrays.asList(entry.getValue().toString().split("\\s*,\\s*"))));
+    } catch (Exception e) {
+      LOGGER.warn("Unable to load tika additional metadata mapping file", e);
```

Comment:
Would it be possible to add the path to the properties file being loaded? That would be helpful to a admin that only has the log file in a prod environment.